modules/appeals_api/app/services/appeals_api/events/appeal_received.rb

```diff
@@ -34,28 +27,58 @@ def vanotify_service
         @vanotify_service ||= VaNotify::Service.new(Settings.vanotify.services.lighthouse.api_key)
       end
 
+      def params(template_id)
+        [
+          lookup,
+          template_id,
+          personalisation
+        ].reduce(&:merge)
+      end
+
+      def lookup
+        if opts['email_identifier'] == 'email'
+          {
+            email_address: opts['email_identifier']['id_value']
```

Comment:
If `opts['email_identifier']` is a string (when it checks for `== 'email'`, how can it also be a hash with a `id_value` key?

Reply:
Good catch